src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -6,9 +6,9 @@
     <RootNamespace>System.Numerics</RootNamespace>
     <DocumentationFile>$(OutputPath)$(MSBuildProjectName).xml</DocumentationFile>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <IsTargetingNetFx Condition="'$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net46'">true</IsTargetingNetFx>
-    <IsTargetingNetCoreApp Condition="'$(TargetGroup)'=='netcoreapp'">true</IsTargetingNetCoreApp>
-    <IsPartialFacadeAssembly Condition="'$(IsTargetingNetFx)'=='true' OR '$(IsTargetingNetCoreApp)'=='true'">true</IsPartialFacadeAssembly>
+    <IsNetFx Condition="'$(TargetGroup)'=='netfx' OR '$(TargetGroup)'=='net46'">true</IsNetFx>
+    <IsNetCoreApp Condition="'$(TargetGroup)'=='netcoreapp'">true</IsNetCoreApp>
+    <IsPartialFacadeAssembly Condition="'$(IsNetFx)'=='true'">true</IsPartialFacadeAssembly>
```

Comment:
I think you still want this to be a partial facade for netcoreapp. While you are adding some types into the assembly other types are still type-forwarded into corelib.

Reply:
This is what is causing your current APICompat errors.